_posts/2021-07-01-origami-2.0-and-how-to-migrate.md

```diff
@@ -0,0 +1,102 @@
+author: Origami
+---
+
+We are have completed our migration of Origami from Bower onto the [NPM registry](https://www.npmjs.com/). The current versions which are on Bower are now deprecated and we recommend all products migrate to the new implementation to keep up-to-date with future releases to Origami.
+
+The [FT Bower Registry](https://origami-bower-registry.ft.com/) has now been placed into a maintenance only mode for 12 months, after then we will look to decommission the service completely.
+
+Moving Origami from Bower to NPM provides many benefits, such as:
+- Aligning our front-end practices with those now most commonly used across the company and the wider industry, which should help the onboarding of new software engineering hires.
+- Origami components can now make use of the 1.6 million packages which are on the NPM registry.
+- Origami components will be able to use modern tooling instead of the Origami team having to build bespoke tooling ourselves.
+- Services such as Snyk, Dependabot, and Renovate will be able to work for the Origami dependencies a project has.
+
+# What has changed for Origami users?
+
+The code for Origami components has remained largely the same, aiming to make the migration simpler for our users. The one change we had to make was making the package name for all components now start with `@financial-times/`, as they are published under the financial-times namespace on npm. This change means that when importing the javascript and/or sass of a component, you will need to prepend `@financial-times/` to the imported package name.
+
+If using Origami components via NPM, it is required to use NPM version 7, this is because version 7 ensures that Origami components are only installed once and no conflicting versions are installed. Origami components do not support using older versions of NPM.
+
+# Plans for the future
+
+Origami used to be versioned only with a major version (Origami 1), this made it hard to track any of the changes being made to Origami which did not warrant incrementing the major version. Now we have incremented the major version, we thought it was a good time to include minors in the versioning scheme. This will make it simpler to make more frequent changes to Origami as a whole and have a record of what thoses changes were and when they were made.
+
+We are looking for people's thoughts on what Origami is missing and/or how Origami could be reimagined to help better achieve it's aim to unify and document the style and experience across the digital products of every FT brand. Some of the parts of Origami we are wanting to improve are around how the documentation and demonstrations for the components are built. Currently they are built in isolation of one another, we would like to see if building them together would lead to a better documented and test set of components.
+
+# First Origami 2.0 Component
```

Comment:
```suggestion
# First npm-only Origami Component
```

Reply:
Thanks, I'll do this as a commit to update the file name and a couple of other references